index.html

```diff
@@ -0,0 +1,146 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Survey Form</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+
+<body>
+
+  <div id="container">
+
+    <header id="header">
+
+      <h1 id="title">freeCodeCamp Survey Form</h1>
+
+      <p id="description">Thank you for taking the time to help us improve the platform</p>
+
+    </header>
+
+  <form id = "survey-form">
+
+      <div class="forms">
+
+        <label id = "name-label" for = "name">Name</label>
+
+        <input type ="text" id = "name" placeholder ="Enter your Name" required></input>
+
+    </div>
+
+    
+    <div class="forms">
```

Comment:
You should use section tags here instead of divs. and all subsequent "forms" divs.